### PR TITLE
Added a login page for future uses

### PR DESCRIPTION
**Pull request by AndersonCeci**
Created a Login Page for future uses 
Screenshot from 2022-12-08 00-13-39

### CODE REVIEW

File: src/app/authentication/authentication.component.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```
Comment (AndiBraimllari) [inline src/app/authentication/authentication.component.html:1]: Drop the `<!DOCTYPE html>`, `html`, `head`, and `body` tags

File: angular.json [hunk 2]
```diff
@@ -110,5 +110,8 @@
       }
     }
   },
-  "defaultProject": "quill"
+  "defaultProject": "quill",
+  "cli": {
+    "analytics": false
+  }
```
Comment (AndiBraimllari) [inline angular.json:116]: Any reason why these three lines are added? If not please drop them

**Review by AndiBraimllari — CHANGES_REQUESTED** [review]

File: angular.json (hunk 2 above)
Comment (AndersonCeci) [reply angular.json]: They were created automatically, I didn't notice them removed on this PR

**Review by AndiBraimllari — APPROVED** [review]

**Comment by AndiBraimllari** [comment]
If you'd like to attempt to improve the indenting in the code you could try running `npm run prettier:fix` (of course after updating your branch locally) and then keep the changes proposed.

**Pull request merged**